sys/conf/kern.mk

```diff
@@ -217,7 +217,7 @@ CFLAGS+=	-fstack-protector
 # but critical for these newer versions.
 #
 .if ${CFLAGS:M-g} != "" && ${CFLAGS:M-gdwarf*} == ""
-CFLAGS+=	-gdwarf-4 -ggdb3
+CFLAGS+=	-gdwarf-2
```

Comment:
I think I would like to see this moved into a .if GCC check of some kind instead.  Don't we lose functionality in newer GCC/CLANG/GDB/LLDB if this is downgraded?